crates/rolldown/tests/esbuild/dce/disable_tree_shaking/artifacts.snap

```diff
@@ -3,14 +3,6 @@ source: crates/rolldown_testing/src/case/case.rs
 expression: content
 input_file: crates/rolldown/tests/esbuild/dce/disable_tree_shaking
 ---
-# warnings
```

Comment:
It seems people tend to ignore warnings. Thinking about should we turn this into errors:

https://github.com/rolldown/rolldown/issues/1174